components/TwitterTestimonials.vue

```diff
@@ -27,8 +27,8 @@ import testimonialsData from "../data/testimonials.json";
 import tweetFixtures from "../fixtures/twitterTweetsApiFixture.json";
 import logger from "../services/logger";
 
-const API_URL =
-  "https://api.twitter.com/2/tweets?user.fields=profile_image_url,username&expansions=author_id&ids=";
+// Axios requires proxies for origin changes. /twitter/ will get overwritten with https://api.twitter.com.
```

Comment:
My initial thought was, rather than doing a proxy, we should enable CORS for the twitter domain in netlify. 
Would there be an advantage to using a proxy over adding twitter to the accepted header @renderf0x ?

Reply:
One good one:

https://twittercommunity.com/t/will-twitter-api-support-cors-headers-soon/28276

Doesn't look like Twitter supports CORS headers.

Reply:
👍